<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alarm</class>
 <widget class="QMainWindow" name="Alarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>321</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New reminder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" alignment="Qt::AlignRight">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Со звуком</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="saveBtn">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Создать напоминание</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>Время:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Добавить сообщение:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignTop">
        <widget class="QTimeEdit" name="timeEdit">
         <property name="displayFormat">
          <string>h:mm </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="chkDate">
         <property name="text">
          <string>Дата:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" rowspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="chkMon">
         <property name="text">
          <string>Пн</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QCheckBox" name="chkWed">
         <property name="text">
          <string>Ср</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QCheckBox" name="chkFri">
         <property name="text">
          <string>Пт</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QCheckBox" name="chkSun">
         <property name="text">
          <string>Вс</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDateEdit" name="dateEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2021</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="chkTues">
         <property name="text">
          <string>Вт</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QCheckBox" name="chkThurs">
         <property name="text">
          <string>Чт</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QCheckBox" name="chkSat">
         <property name="text">
          <string>Сб</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="4">
        <widget class="QTextEdit" name="msgEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="listOfReminders">
   <property name="text">
    <string>Список созданных напоминаний</string>
   </property>
  </action>
  <action name="addQuickReminder">
   <property name="text">
    <string>Создать быстрое напоминание</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
